{"uri":"file:///home/tarcisio/dev/phpfin/vendor/illuminate/database/Eloquent/RelationNotFoundException.php","root":{"kind":0,"name":"","children":[{"kind":512,"name":"Illuminate\\Database\\Eloquent","location":[2,0,2,39]},{"kind":1,"name":"RuntimeException","modifiers":4096,"location":[4,4,4,20],"associated":[{"kind":1,"name":"RuntimeException"}]},{"kind":1,"name":"Illuminate\\Database\\Eloquent\\RelationNotFoundException","location":[6,0,21,1],"associated":[{"kind":1,"name":"RuntimeException"}],"children":[{"kind":32,"name":"make","modifiers":33,"location":[15,4,20,5],"description":"Create a new exception instance.","scope":"Illuminate\\Database\\Eloquent\\RelationNotFoundException","type":"Illuminate\\Database\\Eloquent\\RelationNotFoundException","children":[{"kind":128,"name":"$model","location":[15,32,15,38],"scope":"make","type":"mixed"},{"kind":128,"name":"$relation","location":[15,40,15,49],"scope":"make","type":"string"}]}]}]}}